src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -308,6 +320,42 @@ void AddSection(string kind, ImmutableArray<TaggedText> taggedParts)
             return default;
         }
 
+        private ImmutableArray<TaggedText> GetReturnsDocumentationContent(
+            IEnumerable<ISymbol> symbols,
+            IDictionary<SymbolDescriptionGroups, ImmutableArray<TaggedText>> sections,
+            SemanticModel semanticModel,
+            SyntaxToken token,
+            IDocumentationCommentFormattingService formatter,
+            ISyntaxFactsService syntaxFactsService,
+            CancellationToken cancellationToken)
+        {
+            if (sections.TryGetValue(SymbolDescriptionGroups.ReturnsDocumentation, out var parts))
```

Comment:
I'm not understanding this if -- so sometimes sections does have this and sometimes doesn't? When is one or the other the case?

Reply:
I'm not entirely sure. This was modeled after the existing `GetDocumentationContent` method.